src/open_samus_returns_rando/files/levels/s110_surfaceb.lua

```diff
@@ -299,6 +308,11 @@ function s110_surfaceb.LaunchRidleyDeadCutscene()
   GUI.ForceIdleCinematicControlledOn()
   Game.LaunchCutscene("cutscenes/ridley4/takes/10/ridley410.bmscu")
   s110_surfaceb.SetLowModelsVisibility(false)
+  if Init.sFinalBoss ~= "Ridley" then
+    Blackboard.SetProp("DEFEATED_ENEMIES", "Ridley", "b", true)
```

Comment:
I cannot find a `SetProp` for the other enemies: I assume they are already set by game internal stuff?

Reply:
It depends. Metroids check for a deaths value, the Queen sets a prop, so I did the latter here.